JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -312,6 +312,32 @@ module Microsoft.ApplicationInsights {
         }
 
         /**
+         * Sets the autheticated user id and the account id in this session.
+         * User auth id and account id should be of type string. They should not contain commas, semi-colons, equal signs, spaces, or vertical-bars.
+         *   
+         * @param authenticatedUserId {string} - The authenticated user id. A unique and persistent string that represents each authenticated user in the service.
+         * @param accountId {string} - An optional string to represent the account associated with the authenticated user.
+         */
+        public setAuthenticatedUserContext(authenticatedUserId: string, accountId?: string) {
```

Comment:
I tend to like this pattern because it is succinct. I was familiar with it before knockout came around, so using `observable()` is a property get and `observable(value)` is a property set didn't feel confusing. However, I can certainly see how this can be confusing for someone not familiar with this pattern in other JavaScript frameworks. Anyway, I don't have a strong preference; I just wanted us to consider it before settling on the get/set method pattern.